pkg/server/status.go

```diff
@@ -38,18 +37,24 @@ func (c *criContainerdService) Status(ctx context.Context, r *runtime.StatusRequ
 		Type:   runtime.RuntimeReady,
 		Status: true,
 	}
-	// Use containerd grpc server healthcheck service to check its readiness.
-	resp, err := c.healthService.Check(ctx, &healthapi.HealthCheckRequest{})
-	if err != nil || resp.Status != healthapi.HealthCheckResponse_SERVING {
+
+	if c.client == nil {
```

Comment:
Don't think we'll start `cri-containerd` if client is nil, so maybe we should ignore this case.